patient-care/webapp/server/httpAPI.js

```diff
@@ -0,0 +1,87 @@
+// TODO: can we use Picker for this? (remove cfs:http-methods)
+
+import Fuse from 'fuse.js';
+
+HTTP.methods({
+  "/search/collaborations": function() {
```

Comment:
Please add some comments about how to use this API. Is it intended for "internal use only"? What's the rationale of making a whole API for this?